shared/teams/add-people/container.js

```diff
@@ -63,11 +74,13 @@ const mapDispatchToProps = (dispatch: Dispatch, {navigateUp, routePath, routePro
       navigateAppend([
         {
           props: {
+            addButtonLabel: 'Add',
             allowOwner,
+            headerTitle: 'Add them as:',
             onComplete,
             selectedRole: role,
-            sendNotificationChecked: sendNotification,
-            showNotificationCheckbox: true,
+            sendNotificationChecked: true,
```

Comment:
why is this true always now?

Reply:
Because the design doesn't have the send notification checkbox in anymore -- @cecileboucheron, if we aren't showing the option anymore, would we like the notification to be always on or off?